src/commands/timer.ts

```diff
@@ -0,0 +1,58 @@
+import BaseCommand, { CommandArgs } from "./base_command";
+import { getDebugContext, sendOptionsMessage } from "../helpers/discord_utils";
+import { GameOption, getGuildPreference } from "../helpers/game_utils";
+import _logger from "../logger";
+const logger = _logger("limit");
+
+export default class TimerCommand implements BaseCommand {
+    async call({ message, parsedMessage, gameSessions }: CommandArgs) {
+        const guildPreference = await getGuildPreference(message.guildID);
+        const gameSession = gameSessions[message.guildID];
+        if (parsedMessage.components.length === 0) {
+            guildPreference.resetTimer();
+            if (gameSession) {
+                gameSession.stopTimer();
+                gameSession.timerVal = null;
```

Comment:
This won't persist the option in the db

Reply:
`gameSession` has `resetTimer`?

Reply:
Removed the non-persistent option, thanks.

`resetTimer` was renamed to `resetGuessTimeout` (the reset naming scheme follows from other functions in the file), and it sets the timer value to null, effectively disabling the timer.